MediaRecorder.html

```diff
@@ -469,6 +498,22 @@
         </dl>
       </section>
     </div>
+    <section>
+      <h2>Data handling</h2>
+        <p><dfn data-dfn-type="dfn" data-lt="fires a dataavailable event">
+        Firing a dataavailable event </dfn> with a <a
```

Comment:
I think this should be "fire a blob event", as specializations of "fire an event" typically use the event interface name and not the event type. So the call sites would say "fire a blob event named dataavailable at *target* with its `blob` attribute initialized to *blob*" or some such.